sdks/java/core/pom.xml

```diff
@@ -139,26 +139,6 @@
       <artifactId>beam-sdks-common-runner-api</artifactId>
     </dependency>
 
-    <dependency>
-      <groupId>io.grpc</groupId>
-      <artifactId>grpc-auth</artifactId>
-    </dependency>
-
-    <dependency>
-      <groupId>io.grpc</groupId>
-      <artifactId>grpc-core</artifactId>
-    </dependency>
-
-    <dependency>
-      <groupId>io.grpc</groupId>
-      <artifactId>grpc-netty</artifactId>
-    </dependency>
-
-    <dependency>
-      <groupId>io.grpc</groupId>
-      <artifactId>grpc-stub</artifactId>
-    </dependency>
-
     <!-- grpc-all does not obey IWYU, so we need to exclude from compile
          scope and depend on it at runtime. -->
     <dependency>
```

Comment:
what about these other two. Should they move as well?

(`maven-dependency-plugin` won't complain about them because they're `runtime` deps.)